packages/core/ormconfig.ts

```diff
@@ -0,0 +1,16 @@
+import { SnakeNamingStrategy } from "typeorm-naming-strategies/snake-naming.strategy"
+import { getConnectionOptions } from "./src/db/Connection"
```

Comment:
ormConfig.ts

Reply:
I think it's a convention in TypeORM to name this file in lower-case only

https://github.com/typeorm/typeorm/blob/master/docs/using-ormconfig.md